palette/src/convert.rs

```diff
@@ -1,9 +1,9 @@
+use crate::encoding::Linear;
+use crate::luma::Luma;
+use crate::rgb::{Rgb, RgbSpace};
+use crate::white_point::{WhitePoint, D65};
+use crate::{FloatComponent, Hsl, Hsv, Hwb, Lab, Lch, Limited, Xyz, Yxy};
```

Comment:
My module grouping convention is

```rust
// Standard libs first
use core::...;
use std::...;

// Then third party stuff in one or more groups
use some_lib::...;

// And finally internal imports
use crate::...;
```

Reply:
That was `cargo fmt` since the lines were touching. Fixing that now.

Reply:
I suspected that was the case. 🙂 I forgot to elaborate, but I just meant to show how I usually do it. Formatting that isn't enforced programmatically isn't a big deal.